### PR TITLE
[ticket/16944] Use icon bundle to load Iconify locally

### PR DESCRIPTION
**Pull request by marc1706**
General behavior: Icon bundle is created based on styles and enabled extensions. Enabling extensions or installing new styles will recreate the icon bundle.

Todo:
- [x] Reach consensus that this is a proper way to go
- [x] (Maybe) add ability to disable bundler when allowing 3rd party apps / cookies
- [x] Clean up code
- [x] Add tests for bundler

Checklist:

- [x] Correct branch: master for new features; 3.3.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/master/coding-guidelines.html) and [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

Tracker ticket:

https://tracker.phpbb.com/browse/PHPBB3-16944

### CODE REVIEW

File: phpBB/includes/acp/acp_styles.php [hunk 1]
```diff
@@ -367,6 +367,9 @@ protected function action_uninstall_confirmed($ids, $delete_files)
 		// Clear cache
 		$this->cache->purge();
 
+		// Force rebuild of iconify bundle
+		//$this->iconify_bundler->get_bundle(true);
```
Comment (rxu) [inline phpBB/includes/acp/acp_styles.php:371]: Probably a development leftover :)

**Comment by rxu** [comment]
Does it possibly make some sense to add some specific functional tests?

File: phpBB/styles/prosilver/template/simple_footer.html [hunk 2]
```diff
@@ -49,7 +49,7 @@ <h3 class="alert_title">&nbsp;</h3><p class="alert_text"></p>
 <script src="https://code.iconify.design/1/1.0.0-rc7/iconify.min.js"></script>
```
Comment (vinny) [inline phpBB/styles/prosilver/template/simple_footer.html:49]: Should be updated same as overall_footer?

**Comment by marc1706** [comment]
> Does it possibly make some sense to add some specific functional tests?

I definitely want to still add some tests, yes.

**Review by rubencm — APPROVED** [review]

**Review by Derky — APPROVED** [review]

**Pull request merged**